### ISSUE TITLE
[Accessibility][Usability]A11y_VSCode_StatusBar_Keyboard: Keyboard does not move to notification window as soon as new notification appears when screenreader is ON.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52688]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Issue Type : Usability

**Repro Steps:**
1. Launch VS Code while Screenreader is ON.
2. Wait for any notifications to appear on screen.

**Actual:**
The keyboard focus does not move to the new notification content that appears on the screen. Screenreader only reads the content info of notification but user is not able to perform action on that notification as soon as it appears. 

Users need to "Tab" till the end of content(entire VS code content from present cursor position/focus) to move to the notification area. 

**Expected:**
The keyboard focus should move to notification content as soon as it appears. User can perform any action on that notification (using provided buttons) or close and continue the previous work. 

In this case the focus should move back to the last focused element or place from where the focus was moved to notification window.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
The user will need to move through all the content and controls present on the screen till last to move to notification window. Urgent actions on notifications cannot be taken by the user.

**Attachment For Reference:**
[A11y_VSCode_StatusBar_ScreenReader_Usability.pptx](https://github.com/Microsoft/vscode/files/2127773/A11y_VSCode_StatusBar_ScreenReader_Usability.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
We will NOT move focus to the notification when it appears. There are keybindings to focus the notification area that people can use.
